.travis.yml

```diff
@@ -15,28 +17,32 @@ addons:
       - python-lxml  # because pip installation is slow
       - wkhtmltopdf
 
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+
 env:
-  - VERSION="10.0" LINT_CHECK="1"
-  - VERSION="10.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="10.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - VERSION="10.0" UNIT_TEST="1" LINT_CHECK="0"
+  global:
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: jX9I8iBSTGMGYNzj5zJkh/bvmXmemjqlKpVezZ2QMqaoXHgPPUlktxBwTMlmNeuJ5L9DlDiCqOiXpF41LDmKmjX/e99G15cvNSfQc/jG+exIfo9KSbc6aQSu8cqP+yu1MGTsuuq67Wm8N45+fFJGGsySUbGWFPXnF4MVIiZpqx0=
+
+  matrix:
+  - LINT_CHECK="1"
+  - TRANSIFEX="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="OCA/OCB"
 
 virtualenv:
   system_site_packages: true
 
-before_install:
-  - "export DISPLAY=:910.0"
-  - "sh -e /etc/init.d/xvfb start"
-
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
-  - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
```

Comment:
I removed this line intentionally - I don't see how it provides functionality to the build. Correct me if I'm wrong.